packages/camera/camera_avfoundation/example/ios/RunnerTests/CameraPermissionTests.m

```diff
@@ -8,31 +8,60 @@
 #endif
 @import AVFoundation;
 @import XCTest;
-#import <OCMock/OCMock.h>
+
 #import "CameraTestUtils.h"
+#import "FLTCameraPermissionManager.h"
+#import "FLTPermissionService.h"
```

Comment:
Nit: "managing" and "servicing" for protocol naming as discussed in the [design doc](https://docs.google.com/document/d/1XsaulkJA6_ZSpM7chkQLhQY25sQqhEMiqHMYzw8H85o/edit?resourcekey=0-_cUjF1c0iBvRLKfV-3gK5A&tab=t.0#heading=h.a6hzk7tvmg6z).

Reply:
I changed `FLTPermissionService` protocol name to `FLTPermissionServicing`. Not sure if `FLTCameraPermissionManager.h` should also be renamed as it is not a protocol

Reply:
Oh if it's not a protocol no need to change.